notebooks/datasets/nldas_visualize_rawdata.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>Why do we install these libraries in this notebook and not the others? These should already be baked into the hub. If you want to list the required libraries you can do it at the top of the notebook but it should be consistent and we should avoid installing libraries every execution of this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>